process/project-lifecycle.md

```diff
@@ -1,156 +1,168 @@
-* Independent projects that fit the OpenSSF mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need)
-* Projects commissioned or sanctioned by the OpenSSF, including initial code for OpenSSF WG collaborations, and "experimental" projects
-* Any project that realistically intends to join OpenSSF Incubation in future and wishes to lay the foundations for that
+* Projects must have a minimum of two maintiners with different company affiliations.
+* Projects must be alinged with the OpenSSF mission _and_ either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+* If contributing an existing Project to the OpenSSF, the contribution must undergo license and IP due diligence by the Linux Foundation (LF).
 
-#### Project Graduation Process: Sandbox to Incubating
-
-Incubating Projects are required to undergo due diligence as a part of the process to move from Sandbox to Incubation. Due Diligence is driven by a TAC or parent WG sponsor. Once the diligence is complete and the proposal made, two weeks are allocated for public comment before a TAC vote is called.
 
 #### Submission Forms for Sandbox
 
 1. [Proposal](TBD)
 2. [Legal](TBD)
 
-#### Submission Forms for Sandbox to Incubating
-
-1. [Proposal](TBD)
-2. [Legal](TBD)
 
 ### Incubating
 
-![Incubating](/process/incubation-process.png)
+Incubating projects represent maturing but not fully realized projects. Incubating projects have an established technical vision and working project, but are still refining implementation, discovering (and adapting to) new use cases, and operating as an open source team.
 
-#### Gives - Review and Reporting
-*** Need Text ***
+#### Project Responsibilities
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices (as recommended by the OpenSSF and others)
+* Maintain a point of contact for vulnerability reports
+* Implement, practice, and refine mature software development and release practices such a following a version schema
 
-#### Gets - Benefits
-*** Need Text ***
+#### Project Support
+* Receives guidance on technical direction from TAC
+* Receive a TAC or WG sponsor for guidance as an Incubation project
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
```

Comment:
@dlorenc 's comment on line #112 applies here as well.

Reply:
Resolving; will be addressed by Marketing Committee